lesson_10/solid/solid_app/src/main/java/com/codedifferently/lesson10/library/Library.java

```diff
@@ -9,8 +10,15 @@
 /** Represents a library. */
 public class Library {
```

Comment:
Great attempt at supporting the new requirements, color me impressed! However, I'd like to see you implement a single method that allows you to checked out any type of library asset without having to know whether it's a book, dvd, magazine, or etc. You can do this using the right interface.